Gemfile

```diff
@@ -13,7 +13,5 @@ gem "jquery-rails"
 # Git. Remember to move these dependencies to your gemspec before releasing
 # your gem to rubygems.org.
 
-group :development, :test do
-  # To use debugger
-  gem 'byebug'
-end
+# To use debugger
+gem 'byebug'
```

Comment:
I don't think we want debugger in production, right?

Reply:
This file will only load locally (it's not in the gemspec file list).